features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/SessionEndedMetric.kt

```diff
@@ -30,6 +30,9 @@ internal class SessionEndedMetric(
 
     private val missedEventCountByType = mutableMapOf<MissedEventType, Int>()
 
+    @Volatile
+    private var skippedFramesCount: Int = 0
```

Comment:
You're writing and reading from different threads. Although the impact would be minute as the value reported would be only off by 1 in case of concurrency, you still want to keep the habit of using atomic primitives for multithreaded usage

Reply:
I think ++ needs atomic because it is a compound - without it you could theoretically have lost operations if two threads read the same initial value

Reply:
Thanks guys, you are right, I changed it to `AtomicInteger` for this variable.